dcos_installer/async_server.py

```diff
@@ -358,6 +359,12 @@ def start(cli_options):
     global options
     options = cli_options
 
+    log.error('The DC/OS web installer (--web) is deprecated and will be removed in 1.12\n'
+              'Please refer to CLI DC/OS Installation Guide:\n'
+              'OSS https://docs.mesosphere.com/1.10/installing/oss/custom/cli/\n'
```

Comment:
shouldnt this be Please refer to DC/OS installation guide instead of mentioning cli? also 1.11 link?

Reply:
@amitaekbote replace "deprecated" with "retired" since "deprecated" will shortly imply we'll fix bugs but not enhance it.  "Retired" means we won't fix bugs or enhance it.